unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/States/FirstPerson/BuilderInWorldFirstPersonMode.cs

```diff
@@ -86,6 +90,31 @@ void LateUpdate()
 
     }
 
+    public void MouseClick(int buttonId, Vector3 mouseposition)
```

Comment:
Mouse position doesn't look like is being used here.
```suggestion
    public void MouseClick(int buttonId)
```

Reply:
I'm not sure about the `MouseClick` name. This is handling a mouse click or forcing one? If its handling one maybe it'd be better to name this `OnMouseClick`?

Reply:
The mouse position is not used there but the `BuilderInWorldInputWrapper `event for the `OnMouseClick `passes it as a parameter, I need it in order to be able to attach the event, create 2 events 1 with the `mousePosition `and 1 without it doesn't seem very clean to me but I can implement them if you want.

For the name, I have changed it